data/hai/unfettered 1 alphas.txt

```diff
@@ -0,0 +1,1532 @@
+			choice
+				`	"What about the damage on their ship? How is that relevant to my mission?"`
+					goto information
+				`	"Can you tell me more about what kind of damage it was, on their ship?"`
+			label damage
+			`	"The damage was done by some kind of beam weapon, and a very strong one at that. It looks like it only got partially deflected by the shields, if at all. I wonder what kind of weapon can do that." She looks intrigued, and pauses to think about it, then continues, "That would explain why it's forbidden for us to venture east with jump drives. We lack any hull repair capabilities, and to be honest with you, our hull is not much more resilient than the one of some human ships, like that Auxiliary."`
+			choice
+				`	"They seemed to be using the same weapon against me."`
+					goto logs
+				`	"What can we use that information for?"`
+					goto information
+			label logs
+			`	"Yes, I noticed on the scanner logs. I would advise not relying too heavily on your shields, but they don't seem to have that many of those weapons, so you should be fine. I do not believe they could have designed such weapons by themselves. They probably just stole them from nearby aliens."`
+			choice
+				`	"What about the damage on their ship? How is that relevant to my mission?"`
+					goto information
+				`	"You should have asked me to do an outfit scan, earlier."`
+					to display
+						not "unfettered: scanned auxiliary's outfits"
+				`	"I did well to scan its outfits anyway then, so we have some information to work with."`
+					to display
+						has "unfettered: scanned auxiliary's outfits"
+					goto scanned
+			`	"Maybe, but it could have provoked them further. They apparently like their secrets. I only wanted to know about the structure of the ship, but I'd agree that knowing more about those weapons would be useful. Now, back to the damage on their ship.`
+				goto information
+			label scanned
+			action
+				"unfettered rep: Mera" ++
+			`	She moves her head to the side, surprised, "You... Did? Let me take a look at it then," she asks, and you give her the details. "It seems like this ship was outfitted for speed rather than combat, luckily for you. They are using 2 Tracker Pods, 2 Bullfrogs and one combo of turrets that disrupt shields and slice through hull. I am not much of a strategist so I'll let you decide what to make out of this intel," she says, before taking another look at the logs. And almost jumping in place, "Wait! They have a Systems Core, and plenty of fuel. That must mean that the damage to their ship should have been repaired very quickly. The trail could run cold very quickly in that case, I'd advise you to head east as soon as you can." She stops for a second, unsure what to say, "No, that can't be. Anyway, back to the damage suffered to their ship."`
```

Comment:
```suggestion
			`	She moves her head to the side, surprised, "You... did? Let me take a look at it then," she asks, and you give her the details. "It seems like this ship was outfitted for speed rather than combat, luckily for you. They are using 2 Tracker Pods, 2 Bullfrogs and one combo of turrets that disrupt shields and slice through hull. I am not much of a strategist so I'll let you decide what to make out of this intel," she says, before taking another look at the logs. And almost jumping in place, "Wait! They have a Systems Core, and plenty of fuel. That must mean that the damage to their ship should have been repaired very quickly. The trail could run cold very quickly in that case. I'd advise you to head east as soon as you can." She stops for a second, unsure what to say, "No, that can't be. Anyway, back to the damage suffered to their ship."`
```